bookings/bookings.yaml

```diff
@@ -99,4 +99,248 @@ BookingsWithEntityId:
       422:
         $ref: '/shared/standard-responses.yaml#/422'
       429:
-        $ref: '/shared/standard-responses.yaml#/429'
\ No newline at end of file
+        $ref: '/shared/standard-responses.yaml#/429'
+BookingEntityLinks:
+  get:
+    summary: Get links for a booking
+    tags:
+      - Bookings
+    description: >-
+      Returns an array of URLs for customer-facing actions relating to a booking
+    operationId: GetBookingLinks
+    parameters:
+      - in: path
+        name: bookingId
+        required: true
+        description: The ID of the booking you want to retrieve links for
+        schema:
+          type: string
+    responses:
+      200:
+        description: The booking links were successfully retrieved.
+        content:
+          application/json:
+            schema:
+              $ref: '/bookings/components/booking-links-response.yaml#'
+      401:
+        $ref: '/shared/standard-responses.yaml#/401'
+      403:
+        $ref: '/shared/standard-responses.yaml#/403'
+      404:
+        $ref: '/shared/standard-responses.yaml#/404'
+      422:
+        $ref: '/shared/standard-responses.yaml#/422'
+      429:
+        $ref: '/shared/standard-responses.yaml#/429'
+BookingEntityAssignedAreas:
+  get:
+    summary: Get assigned areas for a booking
```

Comment:
Do you think it's worth making this something like `Get assigned areas (tables) for a booking` so at least in one place we're clarifying that what we refer to as an 'area' is commonly known as a table?